src/psyclone/parse/kernel.py

```diff
@@ -624,29 +633,39 @@ def get_kernel_metadata(name, ast):
         raise ParseError("Kernel type {0} does not exist".format(name))
     return ktype
 
+
 def get_kernel_interface(ast):
     '''Takes the kernel module parse tree and returns the interface part
     of the parse tree.
 
     :param ast: parse tree of the kernel module code
     :type ast: :py:class:`fparser.one.block_statements.BeginSource`
 
-    :returns: Name 'name' of the parse tree of the interface 
-    :rtype: :str `fparser.one.block_statements.Interface.name`
+    :returns: Name of the parse tree of the interface block and
+              the names of the module procedures. Or None, None if
```

Comment:
Once you've made the change I've suggested below this needs to say "... procedures (lower case). Or ..."

Reply:
Strictly also, the parse tree doesn't have a 'name'. Maybe: "Name of the interface block extracted from the parse tree ..."

Reply:
OK. Done.